smoke-tests/tap-snapshots/test/index.js.test.cjs

```diff
@@ -66,7 +66,6 @@ npm error
 npm error aliases: clean-install, ic, install-clean, isntall-clean
 npm error
 npm error Run "npm help ci" for more info
-
```

Comment:
Testing locally this newline is now gone altogether in those other situations. Consistency is the thing here so we're good.

Reply:
Yes, glad you noticed. I couldn't find the true historical reason for this, but in refactoring I'm not confident we always display at least something to the user in all cases (even weird ones like npm is never set and then we error). But since this is truly just a display oriented newline, I wanted to remove it.